kong fails to build on amd64

The package kong fails to build from source with the following error:

```
Hunk #1 succeeded at 50.
done
Hmm...  Looks like a unified diff to me...
The text leading up to this was:
--------------------------
|diff --git a/build/BUILD.bazel b/build/BUILD.bazel
|index b83f4f4..42aa986 100644
|--- a/build/BUILD.bazel
|+++ b/build/BUILD.bazel
--------------------------
patching file build/BUILD.bazel
Using Plan A...
Hunk #1 succeeded at 10.
Hunk #2 succeeded at 20.
Hunk #3 succeeded at 46.
Hunk #4 succeeded at 71.
Hmm...  The next patch looks like a unified diff to me...
The text leading up to this was:
--------------------------
|diff --git a/build/luarocks/BUILD.luarocks.bazel b/build/luarocks/BUILD.luarocks.bazel
|index b23ad96..d9dda79 100644
|--- a/build/luarocks/BUILD.luarocks.bazel
|+++ b/build/luarocks/BUILD.luarocks.bazel
--------------------------
patching file build/luarocks/BUILD.luarocks.bazel
Using Plan A...
Hunk #1 succeeded at 40.
Hmm...  The next patch looks like a unified diff to me...
The text leading up to this was:
--------------------------
|diff --git a/build/luarocks/templates/luarocks_exec.sh b/build/luarocks/templates/luarocks_exec.sh
|index 10515f8..f8cc445 100644
|--- a/build/luarocks/templates/luarocks_exec.sh
|+++ b/build/luarocks/templates/luarocks_exec.sh
--------------------------
patching file build/luarocks/templates/luarocks_exec.sh
Using Plan A...
Hunk #1 succeeded at 3.
Hunk #2 succeeded at 26.
Hunk #3 succeeded at 113.
Hmm...  The next patch looks like a unified diff to me...
The text leading up to this was:
--------------------------
|diff --git a/build/openresty/BUILD.openresty.bazel b/build/openresty/BUILD.openresty.bazel
|index 81ad172..157f6b7 100644
|--- a/build/openresty/BUILD.openresty.bazel
|+++ b/build/openresty/BUILD.openresty.bazel
--------------------------
patching file build/openresty/BUILD.openresty.bazel
Using Plan A...
Hunk #1 succeeded at 147.
Hunk #2 succeeded at 324.
Hmm...  The next patch looks like a unified diff to me...
The text leading up to this was:
--------------------------
|diff --git a/build/openresty/repositories.bzl b/build/openresty/repositories.bzl
|index f4cafe9..6ced994 100644
|--- a/build/openresty/repositories.bzl
|+++ b/build/openresty/repositories.bzl
--------------------------
patching file build/openresty/repositories.bzl
Using Plan A...
Hunk #1 succeeded at 7.
Hunk #2 succeeded at 30.
Hmm...  Ignoring the trailing garbage.
done
Extracting Bazel installation...
Starting local Bazel server (8.0.1-0) and connecting to it...
(06:41:19) [31m[1mFATAL: [0mbazel crashed due to an internal error. Printing stack trace:
java.lang.IllegalStateException: java.lang.RuntimeException: Unrecoverable error while evaluating node 'IGNORED_SUBDIRECTORIES:@@' (requested by nodes )
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.evaluateSkyKeys(SkyframeExecutor.java:2057)
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.evaluateSkyKeys(SkyframeExecutor.java:2033)
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.handleDiffs(SkyframeExecutor.java:3293)
	at com.google.devtools.build.lib.skyframe.SequencedSkyframeExecutor.sync(SequencedSkyframeExecutor.java:293)
	at com.google.devtools.build.lib.runtime.CommandEnvironment.syncPackageLoading(CommandEnvironment.java:779)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:585)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:253)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:607)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$0(GrpcServerImpl.java:677)
	at io.grpc.Context$1.run(Context.java:566)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.RuntimeException: Unrecoverable error while evaluating node 'IGNORED_SUBDIRECTORIES:@@' (requested by nodes )
	at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:547)
	at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor$WrappedRunnable.run(AbstractQueueVisitor.java:435)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1403)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: java.lang.IllegalArgumentException
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
	at com.google.devtools.build.lib.cmdline.IgnoredSubdirectories.of(IgnoredSubdirectories.java:103)
	at com.google.devtools.build.lib.skyframe.IgnoredSubdirectoriesValue.of(IgnoredSubdirectoriesValue.java:45)
	at com.google.devtools.build.lib.skyframe.IgnoredSubdirectoriesFunction.compute(IgnoredSubdirectoriesFunction.java:191)
	at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:467)
	... 7 more
failed to build package: unable to run package kong pipeline: unable to run pipeline: exit status 37
```

--- a/kong.yaml
+++ b/kong.yaml
@@ -1,7 +1,7 @@
 package:
   name: kong
   version: 3.9.0
-  epoch: 2
+  epoch: 3
   description: "The Kong Gateway - an API Gateway built on Nginx and OpenResty"
   copyright:
     - license: Apache-2.0
@@ -13,7 +13,7 @@ environment:
   contents:
     packages:
       - bash
-      - bazel
+      - bazel<8
       - bubblewrap
       - build-base
       - busybox

